app/views/identities/_show_sites.erb

```diff
@@ -2,10 +2,10 @@
   <% @sites.map do |site| %>
```

Comment:
is the file extension correct on this file? don't you even know HTML??

Reply:
great pick up, will correct. but it seems to be working w/o html 🤣